### PR TITLE
chore: weekly dependency update

### PR DESCRIPTION
**Pull request by ekashida**
## Does this PR introduce breaking changes?

No

## GUS work item

W-7258582

### CODE REVIEW

File: packages/@lwc/synthetic-shadow/src/env/event.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const focusEventRelatedTargetGetter: (this: FocusEvent) => EventTarget | null =
+    getOwnPropertyDescriptor(FocusEvent.prototype, 'relatedTarget')!.get!;
+
+export {
+    eventCurrentTargetGetter,
+    eventPhaseGetter,
+    eventTargetGetter,
+    focusEventRelatedTargetGetter,
+};
```
Comment (nolanlawson) [inline packages/@lwc/synthetic-shadow/src/env/event.ts:32]: Is this file related to the dep update?

Comment (ekashida) [reply packages/@lwc/synthetic-shadow/src/env/event.ts:32]: It's not, thanks for catching that.

**Review by pmdartus — APPROVED** [review]

**Review by nolanlawson — APPROVED** [review]
